sdk/keyvault/test-resources.json

```diff
@@ -51,7 +51,26 @@
         }
     },
     "variables": {
-        "azureKeyVaultUrl": "[format('https://{0}.vault.azure.net', parameters('baseName'))]"
+        "azureKeyVaultUrl": "[format('https://{0}.vault.azure.net', parameters('baseName'))]",
+        "mgmtApiVersion": "2019-04-01",
```

Comment:
If these various variables are used more than once, can you include them in-place so it's easier to review impacts of changes, for example?

Reply:
I'm not sure what you mean by "include them in-place". Could you explain, please?

Reply:
I mean, don't reference variables if they are only used once. Just use their values in place of the variables like I did throughout the existing test-resources.json. IMO, makes it easier to read if not using variables to keep otherwise multiple uses consistent.